.github/workflows/maven.yml

```diff
@@ -1,29 +1,22 @@
-name: maven
+name: Java CI with Maven
+
 on:
   push:
-    branches:
-      - master
+    branches: [ "master" ]
   pull_request:
-    branches:
-      - master
+    branches: [ "master" ]
+
 jobs:
-  test:
-    name: Build
-    strategy:
-      matrix:
-        os: [ubuntu-20.04, windows-2022, macos-12]
-        java: [17]
-    runs-on: ${{ matrix.os }}
+  build:
+    runs-on: ubuntu-latest
```

Comment:
@hizmailovich version should be fixed

unit tests should be executed on various platforms, such as ubuntu, macOS, windows

integration tests should be executed only on ubuntu platform, since you need to have docker environment in your build

Reply:
@hizmailovich the example is [here](https://github.com/eo-cqrs/eo-kafka/blob/master/.github/workflows/maven.yml)